documents/data-sharing.md

```diff
@@ -0,0 +1,58 @@
+# Annual survey data sharing
+
+This document describes how data from the annual survey should be shared.
+
+## Objectives
+
+In priority order:
+
+* Protect the privacy of respondents.
+* Provide high quality data (unbiased data which is unlikely to be misinterpreted, c.f., raw data).
+* Provide useful and actionable data to the Rust project (i.e., teams and working groups, contributors, and the Foundation).
+* Provide useful data for users and potential users of Rust and providers of Rust tools and services.
+* Provide data in a timely manner.
+* Provide marketing for Rust.
+* Avoid overloading people working directly with the survey data (note that there were over 9000 completed surveys and multiple opportunities for free form responses).
+
+Non-goals:
```

Comment:
I'm unsure how to word this, but I also believe we should be explicit that the survey is a best attempt and not meant to be a rigorous statistical study. This means we may ask questions that to the trained survey creator and statistician are not necessarily ideal.